picoquic/logger.c

```diff
@@ -1347,9 +1347,9 @@ void picoquic_log_tls_ticket(FILE* F, picoquic_connection_id_t cnx_id,
     uint32_t lifetime = 0;
     uint32_t age_add = 0;
     uint8_t nonce_length = 0;
-    uint8_t* nonce_ptr = NULL;
+    /* uint8_t* nonce_ptr = NULL; */
```

Comment:
Why do you not delete this commented out code? I prefer to delete code that is not used and if you still require it some later time, you have git to get it back :)

Reply:
Yes, you are right. It is just my lack of proficiency with git...